web_src/js/components/RepoContributors.vue

```diff
@@ -0,0 +1,421 @@
+<script>
+import {SvgIcon} from '../svg.js';
+import {
+  Chart,
+  Title,
+  Tooltip,
+  Legend,
+  BarElement,
+  CategoryScale,
+  LinearScale,
+  TimeScale,
+  PointElement,
+  LineElement,
+  Filler,
+} from 'chart.js';
+import {GET} from '../modules/fetch.js';
+import zoomPlugin from 'chartjs-plugin-zoom';
+import {Line as ChartLine} from 'vue-chartjs';
+import 'chartjs-adapter-dayjs-4/dist/chartjs-adapter-dayjs-4.esm';
+import $ from 'jquery';
```

Comment:
Is juquery import necessary since we already import fetch?

Reply:
Yes, because we are using it on line 94:

```js
$('.ui.dropdown').dropdown({
  onChange: (val) => {
    this.xAxisMin = this.xAxisStart;
    this.xAxisMax = this.xAxisEnd;
    this.type = val;
    this.sortContributors();
  }
});
```

Reply:
Fixed in [e8a5c57](https://github.com/go-gitea/gitea/pull/27882/commits/e8a5c577713f6dd2bc38b2208e2a2e20907331d1)